plugins/net_plugin/net_plugin.cpp

```diff
@@ -1095,7 +1095,7 @@ namespace eosio {
       void handle_message( const block_id_type& id, signed_block_ptr ptr );
       void handle_message( const packed_transaction& msg ) = delete; // packed_transaction_ptr overload used instead
       void handle_message( packed_transaction_ptr trx );
-      void handle_message( const chain::hs_message& msg );
+      void handle_message( const chain::hs_vote_message& msg );
```

Comment:
Need to update protocol.hpp

Reply:
Which `protocol.hpp`? The one `include/eosio/chain/protocol.hpp` contains only one line of `#include <eosio/chain/block.hpp>`.

Reply:
[net_plugin](https://github.com/AntelopeIO/leap/blob/d2af8636c116a66d530af34bc3565f73b7ed29eb/plugins/net_plugin/include/eosio/net_plugin/protocol.hpp#L146-L146)

Reply:
`hs_message` is used in a number of places. Will remove it after unification.